### DISCUSSION TITLE
Users from Russia can't pay?

### DISCUSSION THREAD

**Discussion by WofWca** [Glench/ExtPay#232] [General]
Hello again! We want to make an extension to help Russian users circumvent censorship. Is it not possible to them to pay for the extension through ExtPay?

**Comment by Glench** [comment]
They should be able to pay just fine. Do you have some evidence that a user in Russia can't pay?

**Reply by WofWca** [reply]
I tried to pay with my own Russian card, and it said "card declined".

Also, Stripe has a page dedicated to sanctions against Russia, but I couldn't find if it says whether you can pay using Russian cards.

**Reply by Glench** [reply]
Can you please link to that Stripe page? And do you have other cards you can try? We haven't had any complaints about users not being able to pay from Russia.
